lib/php/lib/Thrift/Transport/TSocket.php

```diff
@@ -242,8 +242,10 @@ public function open()
       throw new TException($error);
     }
 
-    $socket = socket_import_stream($this->handle_);
-    socket_set_option($socket, SOL_TCP, TCP_NODELAY, 1);
+    if (function_exists('socket_import_stream') && function_exists('socket_set_option')) {
```

Comment:
In high-traffic situations, this change can create a TIME_WAIT pileup. 0.11.0 is unusable for us because of this issue, please either revert or make it configurable defaulting to off.

Reply:
I'd like to know a little more.  If the functions don't exist they cannot be called.  Is the problem that function_exists is returning the wrong result?

Reply:
My analysis was incorrect, it wasn't your patch causing our issue @jeking3 sorry for jumping the gun.